### DISCUSSION TITLE
Zigbee2MQTT - How to setup AUTOSTART of Z2m when LXC is started?

### DISCUSSION THREAD

**Discussion by aledelgo** [tteck/Proxmox#3398] [General] [answered]
I'm on Debian LXC created by the script. all is working fine but if the machine or LXC is rebooted,  Zigbee2MQTT have to be started manually with "npm start" in the console... 

How to make it start automatically at boot) (and, just for curiosity, is there any tech reason why this feature has not been included in the script for such a software)?

**Comment by tteck** [comment, marked as answer]
It is setup to start on boot
https://github.com/tteck/Proxmox/blob/cfa1b2e04d9056fe96981677bd856f7111ad9495/install/zigbee2mqtt-install.sh#L75

**Reply by aledelgo** [reply]
don't know why the service file has not been created on my LXC. i've created it manually and worked.
